test/index.js

```diff
@@ -67,6 +67,69 @@ var squareRoot = function(n) {
                : S.Right(Math.sqrt(n));
 };
 
+var Identity = function(x) {
```

Comment:
Sure. :zap: I've included signatures for `Compose` and the Arbitrary constructors too.